engine/consensus/approvals/Readme.md

```diff
@@ -126,5 +126,46 @@ Instead, we only want to cache _one_ approval per Verification node for each spe
 `ResultApproval.PartialID()`, which is only computed over the chunk-identifier plus the verifier's node ID.  
 
 
+# Emergency Sealing
+
+Emergency sealing is a temporary fallback to maintain liveness in case there are problems with verification.
+In a nutshell, emergency sealing means that (selected) consensus nodes can seal blocks without sufficient approvals
+from verification nodes. 
+
+Conceptually, emergency sealing follows a similar process as regular sealing: we consider verification assignments that
+have not yet been sealed (or orphaned). We inspect each assignment individually whether it satisfies the emergency sealing
+conditions (details below). If this is the case, we generate a _candidate seal_ and place it in the consensus node's local
+`IncorporatedResultSeals` mempool. The block builder will then pick out a continuous sequence of seals from the mempool
+if such exists. This simplifies the sealing logic (happy path as well as emergency sealing fallback), because we can
+evaluate sealing conditions for each incorporated Execution Result [ER] individually. 
+
+Consider block `A`, whose Execution Result [ER] was incorporated in block `B` (illustrated below). 
+we consider an  for block `B`. The decision whether the ER can be emergency sealed is governed by two protocol
+parameters: `DefaultEmergencySealingThresholdForExecution` and `DefaultEmergencySealingThresholdForVerification`. 
+For an ER to be emergency sealed, all of the following conditions have to be satisfied:
+
+![Emergency sealing](/docs/Emergency_Sealing.png)
+
+1. Let Δh<sub>1</sub> be the height difference between the latest finalized block and block `A`.
+   We require that Δh<sub>1</sub> > `DefaultEmergencySealingThresholdForExecution`.
+   * This means that block `A` must have at least `DefaultEmergencySealingThresholdForExecution` unsealed but
```

Comment:
Maybe we should call it `DefaultEmergencySealingThresholdForFinalization`, because it's the threshold for how many blocks have been finalized after the block, before it can be considered to be emergency sealed. 
```suggestion
   * This means that block `A` must have at least `DefaultEmergencySealingThresholdForFinalization` unsealed but
```